neo4j-hot-backup@.service

```diff
@@ -1,25 +1,26 @@
 [Unit]
 Description=Neo4j hot backup
-After=neo4j@%i.service
-Requires=neo4j@%i.service
```

Comment:
This would require neo4j@1.service to be running, which locks the backup to a particular unit.  and reduces the advantage of clustering. Instead of requiring neo4j to be explicitly running, it will run off whichever Neo is on the machine it's assigned to.